### ISSUE TITLE
Way to access nested JSON data

### ISSUE THREAD

**Issue by rozza-m** [amcolash/MMM-json-feed#1]
At the moment there doesn't seem to be a way to access nested data.

So if the key to be retrieved is "Name", buried in the following JSON:
```

{
    "monitor": {
        "Monitor": {
            "Id": "1",
            "Name": "Test1"
    }
   }
 }
```

...there doesn't seem to be a way to retrieve it?

**Comment by rozza-m** [comment]
Replacing lines 45 - 49 with:

```
        for (var i = 0; i < values.length; i++) {
          var tokens = values[i].split(".");
          var obj = data;
          for (var j = 0; j < tokens.length; j++) {
              obj = obj[tokens[j]];
            }
          tableElement.appendChild(this.addValue(values[i], obj));
        }
```

...fixes this. However I'm having issues where running the module multiple times gets it confused, and I'm not good enough with JS to know if this is my doing or not.

**Comment by amcolash** [comment]
I started this as a very simple project but am happy for the feedback! I can check into this in the next few days when I got some time

**Comment by amcolash** [comment]
Alright, so I just added in some new code to allow nested objects/keys to be accessed. It should be fairly straightforward to use. You will need to update the module, just `cd` to the checkout directory and then `git pull`.

In your example, let's say you want to get the Id/Name of the monitor, you can add the following to your config: `values: ['monitor.Monitor.Id', 'monitor.Monitor.Name']`. This will add each to their own line. Alternatively, if you want them stuck together you could just do `monitor.Monitor`.

I have also added in a config value to strip the parent keys off of a value to make it a bit cleaner. This will allow for the names to print as `Id` and `Name` instead of `monitor.Monitor.Id`, `monitor.Monitor.Name`.

Hopefully this is enough for you to get started with. Check the README for the updated options you can add. If you do need more help/options just make a new issue! Thanks for the feedback.

**Comment by amcolash** [comment]
Implemented with 8f1fe6de1c5c5847a7c9228a53343c2dd0c8fde0

**Comment by rozza-m** [comment]
Awesome! Much appreciated, thanks. I'll try it out asap.
